aries-site/src/themes/aries.js

```diff
@@ -212,6 +332,33 @@ export const aries = {
       xxlarge: '166px',
     },
   },
+  paragraph: {
+    small: {
+      size: '14px',
+      height: '20px',
+      maxWidth: '336px',
+    },
+    medium: {
+      size: '18px',
```

Comment:
This should be bumped up to 24px.

Reply:
We need to make sure this bump/change is derived from the `scale` and `spacing` parameters and not just changed manually (though it could have been much easier :) )